src/clojurians_log/db/queries.clj

```diff
@@ -82,7 +82,10 @@
 
 (defn channel-days [db chan-name]
   (when-let [indexes @!indexes]
-    (let [{:keys [chan-day-cnt chan-name->id] :as index} @!indexes]
+    (let [{:keys [chan-day-cnt chan-name->id] :as index}
+          (if (seq @!indexes)
+            @!indexes
+            {:chan-day-cnt {} :chan-name->id {}})]
```

Comment:
This is a hack to solve #61. I'd like to find a clean solution. But the suggested (defonce !indexes (atom {:chan-name->id {} ,,,})) does not work, because after calling (go) at the repl, !indexes is reset to empty map {}. Question: which code is actually called when I type (go) at the repl?

Reply:
`!indexes` is already dereferenced on line 84, so instead of `@!indexes` just use `indexes`. I should have done that too on line 85. But this doesn't seem like the right solution, the `when-let` means that this function may return `nil` if there are no indexes, so the consumer should be able to work with that.

Reply:
`(go)` calls `(user/go)`, which calls `(reloaded.repl/go)`. This initializes the system (see `user.clj:15`) and starts it, so it starts all the components you see `clojurians-log.application/prod-system` + garden-watcher.

I would approach this differently, and make sure the indexes are `nil` when they are not built yet, or are built but there is no data. so change `clojurians-log.db.queries` so that `(def !indexes (atom nil))`, and on line 26 in the `reduce` replace the empty map with `nil`. This is fine because `(assoc-in nil [...] ...)` will return a map. This means `channel-days` will just work, either it returns a value, or `nil`. Then it's a matter of making sure the views work with that. Having a quick look I think we need an `(if channel-days ...)` branch in `log-page-header`